pkg/dbrole/dbrole.go

```diff
@@ -16,55 +16,98 @@
 // specific language governing permissions and limitations
 // under the License.
 
-//	 DAL uses 4 database roles/users to perform all operations,
+//	DAL uses 4 database roles/users to perform all operations,
 //
-//		- `TENANT_READER` - has read access to its tenant's data
-//		- `READER` - has read access to all tenants' data
-//		- `TENANT_WRITER` - has read & write access to its tenant's data
-//		- `WRITER` - has read & write access to all tenants' data
+//	  - `TENANT_READER` - has read access to its tenant's data
+//	  - `READER` - has read access to all tenants' data
+//	  - `TENANT_WRITER` - has read & write access to its tenant's data
+//	  - `WRITER` - has read & write access to all tenants' data
 //
-//	 DAL allows to map a user's service role to the DB role that will be used for
-//	 that user. If a user has multiple service roles which map to several DB roles,
-//	 the DB role with the most extensive privileges will be used (see `DbRoles()`
-//	 for reference to ordered list of DbRoles.
+// Corresponding *INSTANCE_* roles access is determined by the Instancer's configuration,
+// allowing it to access records exclusively with a specific instance.
```

Comment:
I think it would have been simpler if we explicitly listed all the DB roles

Reply:
Will update it